<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowMain</class>
 <widget class="QMainWindow" name="WindowMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>506</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>506</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>506</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>59</red>
        <green>49</green>
        <blue>75</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>59</red>
        <green>49</green>
        <blue>75</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>59</red>
        <green>49</green>
        <blue>75</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>59</red>
        <green>49</green>
        <blue>75</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>59</red>
        <green>49</green>
        <blue>75</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>59</red>
        <green>49</green>
        <blue>75</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>59</red>
        <green>49</green>
        <blue>75</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>59</red>
        <green>49</green>
        <blue>75</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>120</green>
        <blue>120</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>120</green>
        <blue>120</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>120</green>
        <blue>120</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>59</red>
        <green>49</green>
        <blue>75</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Air Hockey</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../Resources/Main.qrc">
    <normaloff>:/Graphics/Logo_002.png</normaloff>:/Graphics/Logo_002.png</iconset>
  </property>
  <widget class="QWidget" name="Centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>32</horstretch>
     <verstretch>48</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>480</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>320</width>
     <height>480</height>
    </size>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <property name="horizontalSpacing">
     <number>0</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="Menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>31</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="File">
    <property name="title">
     <string>GAME</string>
    </property>
    <addaction name="buttonPlay"/>
    <addaction name="buttonSettings"/>
    <addaction name="actionLEAVE"/>
    <addaction name="buttonExit"/>
   </widget>
   <addaction name="File"/>
  </widget>
  <action name="buttonExit">
   <property name="text">
    <string>EXIT</string>
   </property>
  </action>
  <action name="buttonPlay">
   <property name="text">
    <string>PLAY</string>
   </property>
  </action>
  <action name="buttonSettings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>SETTINGS</string>
   </property>
  </action>
  <action name="actionLEAVE">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>LEAVE</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../Resources/Main.qrc"/>
 </resources>
 <connections/>
</ui>
